conftest.py

```diff
@@ -0,0 +1,20 @@
+"""
```

Comment:
Does this file have to be in the repo root directory?

Reply:
I know it works in `/scico/` or in `/scico/scico/`. In the later case, sphinx was trying to make docs pages for it and that caused errors. I doesn't work in `/scico/test/`.

It's ugly, but I couldn't find a way around it.

Reply:
Strange, the `pytest` docs imply that it should be in the `tests` directory.

Reply:
But that's for normal tests, not doctests. I don't see a way around this either.